generator/nextjs/template/src/components/issuance/Offer.tsx

```diff
@@ -3,8 +3,28 @@ import QRCode from "qrcode.react";
 import { FC } from "react";
 import { OfferPayload } from "src/types/types";
 
+function buildClaimLinkInternal(vaultUrl: string, credentialOfferUri: string): string {
+  const params = new URLSearchParams()
+  params.append('credential_offer_uri', credentialOfferUri)
+  const queryString = params.toString()
+  return `${vaultUrl}/claim?${queryString}`
+}
+
+function getClaimLink(credentialOfferUri: string) {
+  if (typeof window !== 'undefined' && window.localStorage) {
+    const vaultUrl = window.localStorage.getItem('affinidiVaultUrl')
+    if (vaultUrl) {
+      return buildClaimLinkInternal(
+        vaultUrl,
+        credentialOfferUri
+      )
+    }
+  }
+  return VaultUtils.buildClaimLink(credentialOfferUri)
+}
+
```

Comment:
These and the `apiGatewayUrl` are internal changes to run the app on dev environment. Please don't merge them to main as they are not meant to be used by developers. Either remove change the MR target to the `with-env` branch.